### PR TITLE
gst-vaapi: added 12bit 422(Y212) support

### PR DESCRIPTION
**Pull request by FocusLuo**
hevc added 12bit 422(Y212) support

Signed-off-by: Luo Focus <focus.luo@intel.com>

### CODE REVIEW

**Comment by FocusLuo** [comment]
This pull depends on:
vaapi-fits:  PR https://github.com/intel/vaapi-fits/pull/227
gst-vaapi:  PR https://gitlab.freedesktop.org/gstreamer/gstreamer-vaapi/-/merge_requests/390

File: test/gst-vaapi/util.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def get_supported_format_map():
     "P010"  : ("p010-10le", "P010_10LE"),
     "I010"  : ("i420-10le", "I420_10LE"),
     "Y210"  : ("y210", "Y210"),
+    "Y212"  : ("y212-le", "Y212_LE"),
```
Comment (uartie) [inline test/gst-vaapi/util.py:37]: Since #238, this file has moved to lib/gstreamer/vaapi/util.py

**Review by uartie — CHANGES_REQUESTED** [review]
Please rebase since #238

**Comment by FocusLuo** [comment]
All of this pull depends on had been merged:
vaapi-fits: PR #227
gst-vaapi: PR https://gitlab.freedesktop.org/gstreamer/gstreamer-vaapi/-/merge_requests/390

**Pull request merged**